File: perl/src/tests/t_read.pl
#!/usr/bin/perl
#
# $Id:$
#
# perl test driver

use Atrshmlog;

$index = shift @ARGV;

$result = Atrshmlog::attach();

$area = Atrshmlog::get_area();

@r = Atrshmlog::read($area, $index);

print "read : $r[0] : \n";

if ($r[0] == 0) {
    print "read : $r[2] : \n";
}
 
print " \n";

exit (0);

# end of main

File: python/src/tests/t_read.py
#!/usr/bin/python3
#
# Id:
#
# We test a bit of the atrshmlog here.
#
# This is for the first starter, so only the basic things.

import sys
import atrshmlog

index = int(sys.argv[1])

result = atrshmlog.attach()

area = atrshmlog.get_area()

r = atrshmlog.read(area, index)

print('read : ' + str(r[0]) + ' : ')

if r[0] == 0 :
    print('read : ' + str(r[2]) + ' : ')
    

print (' ')

exit(0)

# end of test

